### PR TITLE
More queries

### PR DESCRIPTION
**Pull request by rebel216**
Tested more queries.And transactions , Rollback ,Average etc.
Screenshot 2022-10-20 at 12 11 09 AM
Screenshot 2022-10-19 at 10 08 56 PM
Screenshot 2022-10-19 at 1 13 53 AM
Screenshot 2022-10-20 at 1 26 54 AM

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -15,7 +15,7 @@ UPDATE animals SET species = '';
 
 START Transaction;
 UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
-UPDATE animals SET species = 'pokemon' WHERE species LIKE '';
+UPDATE animals SET species = 'pokemon' WHERE species = "";
```
Comment (mikethreels) [inline queries.sql:18]: - [x] Before and after every query you should Verify that change were made.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Anomaly, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- professional looking readme file

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (rebel216) [reply queries.sql:18]: DONE 👍 

Screenshot 2022-10-20 at 3 06 56 AM

**Review by DeliceLydia — APPROVED** [review]
## Hi @rebel216, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rebel216** [comment]
More queries.


Screenshot 2022-10-21 at 9 29 39 PM
Screenshot 2022-10-21 at 9 29 11 PM
Screenshot 2022-10-21 at 9 28 57 PM

**Pull request merged**